### PR TITLE
Add xpack.apm.enabled to whitelist

### PR DESCRIPTION
**Pull request by sorenlouv**
APM in x-pack currently uses `xpack.apm.ui.enabled` but the plan is to move to `xpack.apm.enabled` for consistency with the other options. Since this is a breaking change we have to keep both until 7.0 before we can remove `xpack.apm.ui.enabled`.

cc @dadoonet 

This is dependent on https://github.com/elastic/x-pack-kibana/pull/4489

### CODE REVIEW

**Comment by tylersmalley** [comment]
@sqren, outside of adding this to the docker whitelist `xpack.apm.enabled` should already work, right?

**Comment by sqren** [comment]
@tylersmalley Ahh, I think you are right. Something just dawned on me - correct me if the following is wrong:
1) there is a global Kibana setting that interprets `xpack.<plugin-name>.enabled` (eg. the plugin doesn't have to actively add support for it.

2) The following is just a hack to hide APM from the menu - not actually disable APM https://github.com/elastic/x-pack-kibana/blob/2acc0ffc5c7733506c59f32b605726560c5b8a06/plugins/apm/index.js#L22-L28

So in short:
 - I should remove `xpack.apm.ui.enabled` from this PR
 - I should close this PR https://github.com/elastic/x-pack-kibana/pull/4489

**Comment by sqren** [comment]
`xpack.apm.ui.enabled` was something we added for Cloud, so they can disable APM plugin without triggering a restart of the server.

**Comment by tylersmalley** [comment]
@sqren, yup that's exactly correct. This PR LGTM and https://github.com/elastic/x-pack-kibana/pull/4489 can be closed since Kibana handles it internally.

**Review by tylersmalley — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
LGTM. Thanks for the help and persistence here.

**Comment by jarpy** [comment]
6.1 c918997
6.2 e63704c
6.x fc7d358
